### PR TITLE
Update validation_trace.go

### PR DESCRIPTION
**Pull request by continuum-ajay-sahani**
Compilation error fixed

type TraceValidationFinishFunc = TraceQueryFinishFunc

change to:
type TraceValidationFinishFunc TraceQueryFinishFunc

### CODE REVIEW

**Comment by vetcher** [comment]
what's golang version you use?

**Comment by pavelnikolov** [comment]
What problem does this solve?

**Comment by pavelnikolov** [comment]
Is this related to #231 ?

**Comment by vetcher** [comment]
@pavelnikolov definitely

**Comment by pavelnikolov** [comment]
What version of Go are you using?

**Comment by continuum-ajay-sahani** [comment]
Go version 1.9, I was getting the compilation error.

File: trace/validation_trace.go [hunk 1]
```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/graph-gophers/graphql-go/errors"
 )
 
-type TraceValidationFinishFunc = TraceQueryFinishFunc
+type TraceValidationFinishFunc TraceQueryFinishFunc
```
Comment (pavelnikolov) [inline trace/validation_trace.go]: TBH, `TraceValidationFinishFunc` is only ever used in this file. IMHO, we don't even need type aliasing here. I'd be happy to merge your PR but I think that the approach is wrong. Instead, we need to remove this line altogether and replace `TraceValidationFinishFunc` on lines 10 and 15 with `TraceQueryFinishFunc`.

Comment (pavelnikolov) [reply trace/validation_trace.go]: WDYT?

**Comment by pavelnikolov** [comment]
I'm closing this PR due to inactivity and I believe that it is no longer relevant after the tracing refactoring.

**Pull request closed without merging**